cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -0,0 +1,131 @@
+{% extends "templates/base_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/text-indent/_macro.njk" import onsTextIndent %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+
+{% block header_area %}
+    <div class="analysis-header {% if page.updates %}analysis-header--flush{% endif %}">
```

Comment:
note: we'll want page type specific classes, or common ones as it was done in #55
One for the front-end task. cc @helenb